git/git.go

```diff
@@ -44,7 +44,7 @@ func GetVersions(ctx context.Context, pckg *packages.Package, packageGitcache st
 		gitVersions = append(gitVersions, Version{
 			Tag:       tag,
 			Version:   strings.TrimPrefix(tag, "v"),
-			TimeStamp: packages.GitTimeStamp(ctx, packageGitcache, tag),
+			TimeStamp: packages.GitTimeStamp(ctx, gitpath, tag),
```

Comment:
`GitTimeStamp` will fail when giving a remote URL and "spam" the log. Can we handle that in a nicer way?

Reply:
+ calling `GetMostRecentVersion` is just waste of time, as `TimeStamp` is `time.Unix(0, 0)`